docs/04-operation-guides/troubleshooting/smsh-03-istio-no-sidecar.md

```diff
@@ -2,9 +2,14 @@
 title: Issues with Istio sidecar injection
 ---
 
+## Symptom & cause
 
-Kyma has sidecar injection enabled by default - a sidecar is injected to every Pod in a cluster without the need to add any labels. For more information, read [this document](#details-sidecar-proxy-injection).
-If a Pod doesn't have a sidecar and you did not disable sidecar injection on purpose, follow these steps to troubleshoot:
+Kyma has sidecar injection enabled by default - a sidecar is injected to every Pod in a cluster without the need to add any labels. For more information, read [this document](../operations/smsh-01-istio-disable-sidecar-injection.md).
+If a Pod doesn't have a sidecar and you did not disable sidecar injection on purpose, you may want to fixt that.
+
+## Remedy
+
+Follow these steps to troubleshoot:
 
 1. Check if sidecar injection is disabled in the Namespace of the Pod. Run this command to check the `istio-injection` label:
```

Comment:
Line 20:   If the command returns `disabled` `,` the sidecar injection is disabled in this Namespace.

^ missing comma


Line 22:  >**WARNING:** Removing the `istio-injection=disabled` label from `the` Namespace results in...

^ missing article